desktop-app/src/store/index.ts

```diff
@@ -190,6 +190,13 @@ const schema = {
     },
     default: [],
   },
+  bookmarks: {
+    type: 'array',
+    items: {
+      type: 'object',
```

Comment:
I think we can be a bit more specific here, to declare the `id`, `name` and `address` field as `properties` of the object.